scr/src/main/java/org/apache/felix/scr/impl/BundleComponentActivator.java

```diff
@@ -270,6 +270,21 @@ protected void initialize(List<ComponentMetadata> cachedComponentMetadata)
                 URL[] descriptorURLs = findDescriptors(m_bundle, descriptorLocation);
                 if (descriptorURLs.length == 0)
                 {
+                    if (descriptorLocation.contains("*")) {
+                        // 112.4.1 The last component of each path in the Service-Component header may
+                        // use wildcards so that Bundle.findEntries can be used to locate the XML
+                        // document within the bundle and its fragments. For example:
+                        //
+                        // Service-Component: OSGI-INF/*.xml
+                        //
+                        // A Service-Component manifest header specified in a fragment is ignored by
+                        // SCR. However, XML documents referenced by a bundle's Service-Component
+                        // manifest header may be contained in attached fragments.
+
+                        // in case of such wildcard, finding nothing does not mean an error (because it
+                        // *might* be found in a fragment that is attached later.
+                        continue;
```

Comment:
Perhaps we should log as TRACE here?  By default that should get ignored since the default log level is WARN.

Reply:
Done!